frontend/packages/console-dynamic-plugin-sdk/scripts/create-pkg.ts

```diff
@@ -0,0 +1,26 @@
+import * as fs from 'fs';
+import * as readPkg from 'read-pkg';
+
+const PKG_NAME = '@openshift-console/dynamic-plugin-sdk';
```

Comment:
Eventually we'll want to make sure that we omit `dynamic` from the package name. `plugin-sdk` is sufficient`

Reply:
I agree. Once we acquire @openshift acc on npmjs, I think the package name should be @openshift/console-plugin-sdk